sdk/python/testvectors/nem/account_key_link.py

```diff
@@ -0,0 +1,34 @@
+from symbolchain import nc
+
+
+def generate(factory):
+	descriptors = {
+		'AccountKeyLinkTransaction_1': {
```

Comment:
why not list instead of dict _here_ and auto generate key names in comprehension ?

Reply:
as explained on discord, names match existing ones in testvectors and didn't want to alter them